python/deltalake/table.py

```diff
@@ -51,6 +52,59 @@
 MAX_SUPPORTED_WRITER_VERSION = 2
 
 
+class Compression(Enum):
+    UNCOMPRESSED = "UNCOMPRESSED"
+    SNAPPY = "SNAPPY"
+    GZIP = "GZIP"
+    BROTLI = "BROTLI"
+    LZ4 = "LZ4"
+    ZSTD = "ZSTD"
+    LZ4_RAW = "LZ4_RAW"
+
+    @classmethod
+    def __from_str__(cls, value: str) -> "Compression":
```

Comment:
is this an "official" dunder method? if not, could it be just `from_str`?

Reply:
It's not, will change it